packages/react-components/src/components/SvgExclamationIcon/SvgExclamationIcon.tsx

```diff
@@ -0,0 +1,20 @@
+// This icon is available as a plain SVG at src/assets/icon-exclamation.svg
+export default function SvgExclamationIcon({ id = "Exclamation icon" }) {
+  return (
+    <svg
+      id={id}
+      width="20"
+      height="20"
+      viewBox="0 0 20 20"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g id="20px/Exclamation icon">
```

Comment:
Can you please remove this `id` attribute and the one on `<path>` below, please? This is essentially just cruft from Figma or whatever app generated the SVG code and isn't used for anything, but attributes like this can conflict with genuine `id` attributes in the rest of the DOM. It should be removed from the SVG too.

Reply:
Fixed in [3bbe419](https://github.com/bcgov/design-system/pull/400/commits/3bbe4191673f67eb26244096c4e8a77d27f12915)!